Server/src/routes/init.route.ts

```diff
@@ -29,15 +30,9 @@ export class InitRoute {
 	 * The constructor to connect all of the routes with specific functions.
 	 */
 	private constructor() {
-		this.router.get("/app/track/:trackId", (req, res) => {
-			return this.getForTrack(req, res)
-		})
-		this.router.get("/app/tracks", (req, res) => {
-			return this.getAllTracks(req, res)
-		})
-		this.router.put("/app", jsonParser, (req, res) => {
-			return this.getTrackByPosition(req, res)
-		})
+		this.router.get("/app/track/:trackId", please_dont_crash(this.getForTrack))
```

Comment:
Could you test if `this` is properly bound in `getForTrack` (js is weird). I suspect that this route won't work in this state, as this would be undefined. (I don't know why).
But at least it should not crash the server

Reply:
It might be possible to solve this using `please_dont_crash(this.getForTrack.bind(this))`, but I am not really sure. The "lambda function" way will work to bind `this` in any case, and the lambda function can also be wrapped in `please_dont_crash`.